.github/workflows/build-preview.yml

```diff
@@ -61,32 +58,39 @@ jobs:
-        export MAS_URL="${{ steps.prepare.outputs.base-url }}mas-${{ steps.prepare.outputs.prid }}.zip"
-        export EXE_URL="${{ steps.prepare.outputs.base-url }}${{ steps.prepare.outputs.prid }}.exe"
-        export STATUS_DATA="{\"state\": \"success\", \"description\": \"Download it here:\", \"context\": \"Linux Preview Build\", \"target_url\": \"${APPIMAGE_URL}\"}"
+        # URLs for API connection and uploads
+        export GITHUB_API_URL="https://api.github.com/repos/deltachat/deltachat-desktop/statuses/${{ github.event.after }}"
+        export APPIMAGE_URL="https://download.delta.chat/desktop/preview/deltachat-desktop-${{ steps.prepare.outputs.prid }}.AppImage"
+        export MAS_URL="https://download.delta.chat/desktop/preview/deltachat-desktop-mas-${{ steps.prepare.outputs.prid }}.zip"
+        export EXE_URL="https://download.delta.chat/desktop/preview/deltachat-desktop-${{ steps.prepare.outputs.prid }}.exe"
+        # Post AppImage download link to check details
+        export STATUS_DATA="{\"state\": \"success\", \
+                             \"description\": \"Download it here:\", \
+                             \"context\": \"Linux Preview Build\", \
+                             \"target_url\": \"${APPIMAGE_URL}\"}"
         curl -f -s -X POST --header "authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" --url "$GITHUB_API_URL" --header "content-type: application/json" --data "$STATUS_DATA"
-        export STATUS_DATA="{\"state\": \"success\", \"description\": \"Download it here:\", \"context\": \"MacOS Preview Build\", \"target_url\": \"${MAS_URL}\"}"
+        # Post MacOS download link to check details
+        export STATUS_DATA="{\"state\": \"success\", \
+                             \"description\": \"Download it here:\", \
+                             \"context\": \"MacOS Preview Build\", \
+                             \"target_url\": \"${MAS_URL}\"}"
         curl -f -s -X POST --header "authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" --url "$GITHUB_API_URL" --header "content-type: application/json" --data "$STATUS_DATA"
-        export STATUS_DATA="{\"state\": \"success\", \"description\": \"Download it here:\", \"context\": \"Windows Preview Build\", \"target_url\": \"${EXE_URL}\"}"
+        # Post Windows download link to check details
+        export STATUS_DATA="{\"state\": \"success\", \
+                             \"description\": \"Download it here:\", \
+                             \"context\": \"Windows Preview Build\", \
+                             \"target_url\": \"${EXE_URL}\"}"
         curl -f -s -X POST --header "authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" --url "$GITHUB_API_URL" --header "content-type: application/json" --data "$STATUS_DATA"
+        if md5sum -c pkgsum --status; then 
+            export STATUS_DATA="{\"state\": \"failed\", \  # this value is probably not valid
```

Comment:
`# this value is probably not valid`
resolve this comment before merge?

Reply:
I don't have time before next week, but yes, I will resolve this.